Assets/MRTK/Examples/Demos/EyeTracking/Scenes/EyeTrackingDemo-01-BasicSetup.unity

```diff
@@ -1308,7 +1322,7 @@ PrefabInstance:
         type: 3}
       propertyPath: m_isInputParsingRequired
       value: 1
-      objectReference: {fileID: 0}
+            objectReference: {fileID: 0}
```

Comment:
this indentation seems weird, but i'm not sure i know enough to be certain one way or another!